UefiTestingPkg/AuditTests/PagingAudit/README.md

```diff
@@ -1,60 +1,61 @@
 # Paging Audit
 
-## SmmPagingAudit
+## SMM Paging Audit
 
-SMM is a privileged mode of the ia32/x64 cpu architecture.  In this environment nearly all system state can
+SMM is a privileged mode of the IA32/X64 cpu architecture.  In this environment nearly all system state can
 be inspected including that of the operating system, kernel, and hypervisor.  Due to it's
 capabilities SMM has become an area of interest for those searching to exploit the system.
 To help minimize the interest and impact of an exploit in SMM the SMI handlers should operate
 in a least privileged model.  To do this standard paging can be leveraged to limit the SMI
 handlers access.  Tianocore has a feature to enable paging within SMM and this tool helps confirm
 the configuration being used.  This tool requires three parts to get a complete view.
 
-### SMM
+### SMM Driver
 
-The SMM driver must be included in your build and dispatched to SMM before the End Of Dxe.  It is
+The SMM driver must be included in your build and dispatched to SMM before the end of DXE.  It is
 recommended that this driver should only be used on debug builds as it reports the entire
 SMM memory environment to the caller.  The shell app will communicate to the SMM driver and
 request critical memory information including IDT, GDT, page tables, and loaded images.
 
-### SMM Version App
+### SMM Shell App
 
 The UEFI shell application collects system information from the DXE environment and then
 communicates to the SMM driver/handler to collect necessary info from SMM.  It then
 writes this data to files and then that content is used by the windows scripts.

```

Comment:
Maybe "data to files. Then files can be decided by windows scripts" to make it clear that it's another step?